tests/test_tga.py

```diff
@@ -30,6 +30,33 @@ def test_inference(self):
             upgraded = TorchGeneratorAgent.upgrade_opt({'beam_size': 5})
             self.assertEqual(upgraded['inference'], 'beam')
 
+    def test_no_greedy_largebeam(self):
+        """
+        Ensures that --beam-size > 1 and --inference greedy causes a failure.
+        """
+        testing_utils.download_unittest_models()
+
+        # we should have an exception if we mix beam size > 1 with greedy
+        with self.assertRaises(ValueError):
+            testing_utils.display_model(
+                dict(
+                    task='integration_tests:multipass',
+                    model_file='zoo:unittest/transformer_generator2/model',
+                    beam_size=5,
+                    inference='greedy',
+                )
+            )
+
+        # and we shouldn't if we have bug
```

Comment:
i don't understand this comment, "we shouldn't if we have bug". do you mean "we shouldn't if we have beam"?